frontend/src/metabase/dashboard/components/DashboardMoveModal.jsx

```diff
@@ -35,6 +35,7 @@ class DashboardMoveModalInner extends React.Component {
                   collectionId={destination.id || ROOT_COLLECTION.id}
                 />
               ),
+              undo: false,
```

Comment:
Remove undo actions because it's not working.
![image](https://user-images.githubusercontent.com/1937582/177955726-68f22aa8-6d65-4e9f-8ef3-2f8da1180f45.png)